server/db/recreate-schema.sql

```diff
@@ -1,10 +1,44 @@
 -- Drop tables in case they already exist
+
+DROP TABLE if exists work_experience;
+DROP TABLE if exists education;
+DROP TABLE if exists profiles;
 DROP TABLE if exists users;
 
+
 -- Create tables
 CREATE TABLE users (
   id SERIAL PRIMARY KEY,
   email    VARCHAR(200) NOT NULL,
   role     VARCHAR(15) NOT NULL,
   password VARCHAR(200) NOT NULL
 );
+
+CREATE TABLE profiles (
+  id  SERIAL PRIMARY KEY,
+  profile_public_id  INT REFERENCES users(id),
+  applicant_id       VARCHAR(200) NOT NULL,
+  location           VARCHAR(100) NOT NULL,
+  bio                VARCHAR(200) NOT NULL,
+  job_title          VARCHAR(50) NOT NULL
```

Comment:
On the NOT NULL: can the location, bio and job_title be null or are they all mandatory fields? Maybe a question for our PM :)